presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/PlanMatchPattern.java

```diff
@@ -133,6 +146,21 @@ public boolean isTerminated()
         return sourcePatterns.isEmpty();
     }
 
+    public static FunctionCall functionCall(String name, String... args)
```

Comment:
This should probably return a matcher. Eventually, we're going to have to assert that arguments match more complex patterns, including nested function calls, operators, etc, so that would be the way to compose them.

Also, it'd be cleaner to introduce matchers for argument names instead of relying on a magic name.

So, for instance, you'd do something like `functionCall("foo", name("a"), name("b"), any(), name("c"))`.